models/articles.models.js

```diff
@@ -1,15 +1,27 @@
-const db = require("../db/connection")
-
+const db = require("../db/connection");
 
 exports.selectArticlesById = (article_id) => {
-    return db
+  return db
     .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
     .then((result) => {
-        if(result.rows.length === 0){
-            return Promise.reject({status: 404, msg: 'article does not exist'});
-        }
-        return result.rows;
-    })
-   
-}
+      if (result.rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "article does not exist" });
+      }
+      return result.rows;
+    });
+};
 
+exports.selectAllArticles = () => {
+    
+    const sqlString = `SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url,
+    COUNT(comments.article_id) AS comment_count  
```

Comment:
have a think how you can cast this as a number rather than a string.

Reply:
CAST (COUNT(comments.article_id AS INTEGER) AS comment_count